packages/playwright-core/src/server/chromium/protocol.d.ts

```diff
@@ -854,7 +854,7 @@ Explainer: https://github.com/WICG/attribution-reporting-api
     }
     /**
      * Details for issues about documents in Quirks Mode
-or Limited Quirks Mode that affects page layouting.
+or Limited Quirks Mode that affects page layout.
```

Comment:
This file is generated from https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/third_party/blink/public/devtools_protocol/browser_protocol.pdl;l=783?q=%22Quirks%20Mode%20that%20affects%20page%20layouting%22%20-f:gen%20-f:Debug&ss=chromium